test/parallel/test-timers-active.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const active = require('timers').active;
+
+// active() should not create a timer for these
+var legitTimers = [
+  { _idleTimeout: 0 },
+  { _idleTimeout: 1 }
+];
+
+legitTimers.forEach(function(legit) {
+  active(legit);
+  // active() should mutate these objects
+  assert.notDeepEqual(Object.keys(legit), ['_idleTimeout']);
```

Comment:
@thefourtheye @Fishrock123 OK, I've improved the tests quite a bit, I think. Looks good to you now?